MdePkg/Include/IndustryStandard/PciExpress21.h

```diff
@@ -268,7 +277,7 @@ typedef union {
     UINT32    NoRoEnabledPrPrPassing                        : 1;
     UINT32    LtrMechanism                                  : 1;
     UINT32    TphCompleter                                  : 2;
-    UINT32    LnSystemCLS                                   : 2;
+    UINT32    Reserved                                      : 2;

```

Comment:
LnSystemCLS is not used any more?

Reply:
Yes, Lightweight Notification has been deprecated and is not in use anymore.